Dockerfile

```
@@ -1,30 +1,45 @@
-FROM ubuntu:16.04
+FROM ubuntu:rolling
 
-RUN apt-get update -qqy && apt-get install -y curl xvfb chromium-browser firefox
-
-RUN ln -s /usr/bin/chromium-browser /usr/bin/google-chrome
-
-RUN apt-get update -qqy \
-  && apt-get -qqy install libnss3 libnss3-tools libfontconfig1 wget ca-certificates apt-transport-https inotify-tools \
-  && rm -rf /var/lib/apt/lists/* /var/cache/apt/*
-
-
-RUN apt-get update -qqy && apt-get -y install ruby-full zlib1g-dev libffi-dev gcc make git
-
-RUN apt-get autoclean
+ENV HEADLESS true
 
-RUN gem install 'watir' --no-ri --no-rdoc
-RUN gem install 'headless' --no-ri --no-rdoc
-RUN gem install 'webdrivers' --no-ri --no-rdoc
-RUN gem install 'watir-scroll' --no-ri --no-rdoc
-RUN gem install 'rspec' --no-ri --no-rdoc
-RUN gem install 'watir-rspec' --no-ri --no-rdoc
-RUN gem install 'webdriver-highlighter' --no-ri --no-rdoc
-RUN gem install 'colorize' --no-ri --no-rdoc
-RUN gem install 'faraday' --no-ri --no-rdoc
-RUN gem install 'rubocop' --no-ri --no-rdoc
+# Verwende die letzte Chromium-Version.
+# => https://launchpad.net/~canonical-chromium-builds/+archive/ubuntu/stage 
+RUN add-apt-repository ppa:canonical-chromium-builds/stage && \
```

```
Step 1/8 : FROM ubuntu:rolling
 ---> b977ae81df17
Step 2/8 : ENV HEADLESS true
 ---> Running in 3e7ebd251fbf
Removing intermediate container 3e7ebd251fbf
 ---> 1a3dc00bad7d
Step 3/8 : RUN add-apt-repository ppa:canonical-chromium-builds/stage &&     apt-get update -qy &&     apt-get install -qy         apt-transport-https         ca-certificates         chromium-browser         curl         firefox         gcc         git         inotify-tools         libffi-dev         libfontconfig1         libnss3         libnss3-tools         make         ruby-full         wget         xvfb         zlib1g-dev &&     apt-get autoclean &&     rm -rf /var/lib/apt/lists/* /var/cache/apt/*
 ---> Running in ee649658ecec
/bin/sh: 1: add-apt-repository: not found
```

---

`add-apt-repository` ist in `software-properties-common` drin. noch als paket mit installieren und dann vor dem autoclean `add-apt-repository xyz` ausführen

---

okay, hab ich übersehen, dann halt so:
```suggestion
RUN apt-get update -qy && \
    apt-get install -y software-properties-common && \
    add-apt-repository ppa:canonical-chromium-builds/stage && \
```